atest/robot/keywords/async_tests.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Suite Setup      Run Tests    ${EMPTY}    keywords/async_keywords.robot
```

Comment:
`async_tests.robot` should be renamed to `async_keywords.robot`. We are running keywords, not tests, asynchronously and that name would also match the file under `atest/testdata`.